vitrina/datasets/templates/vitrina/datasets/detail.html

```diff
@@ -21,6 +21,11 @@
 
 {% block content %}
     <div class="columns no-margin-bottom">
+        {% if user.is_authenticated and user == dataset.manager %}
```

Comment:
```suggestion
        {% if can_update_dataset %}
```

Add `can_edit_dataset` to template and get value for it from a service function of the same name. Service function should return

```python
def can_update_dataset(user: User, dataset: Dataset) -> bool:
    return user.is_authenticated and (
        user.is_staff or
        user.organization == dataset.organization or
        user == dataset.manger
    )
```